src/Common/Admin/Entities/Heading.php

```diff
@@ -77,9 +83,10 @@ public function render() {
 		 * 3. The tag_escape() function (which should be used for escaping HTML tags) operates on strings, not integers.
 		 */
 		printf(
-			'<h%1$d class="%2$s">%3$s</h%1$d>',
+			'<h%1$d class="%2$s" %3$s>%4$s</h%1$d>',
 			$this->level, // phpcs:ignore StellarWP.XSS.EscapeOutput,WordPress.Security.EscapeOutput.OutputNotEscaped
 			esc_attr( $this->get_classes() ),
```

Comment:
The reason I've avoided using `tribe_classes()` is because it creates a new instance of `Element_Classes` when it is called, and simply calls the `get_attribute()` method. With that said, we could use `Element_Classes::get_attribute()` directly to prevent an empty attribute.

In short, I like the simplicity of `tribe_classes()`, but its simplicity is also inflexible.

Reply:
Fair.